go.mod

```diff
@@ -1,6 +1,8 @@
 module code.gitea.io/gitea
 
-go 1.23
+go 1.23.0
+
+toolchain go1.23.6
```

Comment:
I don't know why go wants to introduce this `toolchain` here, but it seems like a golang bug to me. See https://github.com/golang/go/issues/65847#issuecomment-2685123221 for my findings.

Reply:
Likely the change itself is bening, `toolchain` is only a "suggestion" to the go compiler.